molo/commenting/tests/test_views.py

```diff
@@ -265,3 +268,36 @@ def test_view_replies_report(self):
         self.assertTrue('report' in crow.prettify())
         self.assertTrue(reply.comment in replyrow.prettify())
         self.assertFalse('report' in replyrow.prettify())
+
+
+@override_settings(ROOT_URLCONF='molo.commenting.tests.test_views')
+class ViewNotificationsRepliesOnCommentsTest(TestCase, MoloTestCaseMixin):
```

Comment:
These tests need to be seperated out into `test_models.py` and `test_views.py`

`test_models.py`
Create a parent comment and a reply (not a notification) and then confirm that the notification has been created, with the correct attributes.

`test_views.py`
If you are testing views, you need to make a request for a page and then look and see if the stuff that you want is in the actual page. This usually involves using `client.request.get("/commenting/...")`

In this case we want a user to

1. Create the users, article, comment and reply in `setUp`
1. Login as a user
1. Request the main page and confirm that the notification is there
1. Get the link from the notification references and request it
1. Check that the comment appears
1. Make another reply
1. Repeat the process and check that there is an unread and read notification in the notifications page